docs/data-cookbooks/sips-workflow.md

```diff
@@ -97,41 +97,53 @@ Finally, let's create a [rule](data-cookbooks/setup.md#rules). In this example w
+_An example workflow module configuration can be viewed in the Cumulus source for the [discover_and_queue_pdrs_workflow](https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/discover_and_queue_pdrs_workflow.tf)._
+
+_**Please note:** To use this example workflow module as a template for a new workflow in your deployment the `source` key for the workflow module would need to point to a release of the cumulus-workflow (terraform-aws-cumulus-workflow.zip) module on our [release](https://github.com/nasa/cumulus/releases) page, as all of the provided Cumulus workflows are internally self-referential._
 
 ## ParsePdr Workflow
 
 The ParsePdr workflow will parse a PDR, queue the specified granules (duplicates are handled according to the duplicate handling setting) and periodically check the status of those queued granules. This workflow will not succeed until all the granules included in the PDR are successfully ingested. If one of those fails, the ParsePdr worklfow will fail. **NOTE** that ParsePdr may spin up multiple IngestGranule workflows in parallel, depending on the granules included in the PDR.
 
-1. ParsePdr - [npm package](https://www.npmjs.com/package/@cumulus/parse-pdr), [source](https://github.com/nasa/cumulus/tree/master/tasks/parse-pdr)
-2. QueueGranules - [npm package](https://www.npmjs.com/package/@cumulus/queue-granules), [source](https://github.com/nasa/cumulus/tree/master/tasks/queue-granules)
-3. CheckStatus - [npm package](https://www.npmjs.com/package/@cumulus/pdr-status-check), [source](https://github.com/nasa/cumulus/tree/master/tasks/pdr-status-check)
+The lambdas below are included in the `cumulus` terraform module for use in your workflows:
+
+1. ParsePdr - [source](https://github.com/nasa/cumulus/tree/master/tasks/parse-pdr)
+2. QueueGranules - [source](https://github.com/nasa/cumulus/tree/master/tasks/queue-granules)
+3. CheckStatus - [source](https://github.com/nasa/cumulus/tree/master/tasks/pdr-status-check)
 
 ![Screenshot of execution graph for SIPS Parse PDR workflow in AWS Step Functions console](assets/sips-parse-pdr.png)
 
-_Example configuration for this workflow can be found in the `ParsePdr` object defined in Cumulus core's [example](https://github.com/nasa/cumulus/blob/master/example/workflows/sips.yml)_
+_An example workflow module configuration can be viewed in the Cumulus source for the [parse_pdr_workflow](https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/parse_pdr_workflow.tf)._
+
+_**Please note:** To use this example workflow module as a template for a new workflow in your deployment the `source` key for the workflow module would need to point to a release of the cumulus-workflow (terraform-aws-cumulus-workflow.zip) module on our [release](https://github.com/nasa/cumulus/releases) page, as all of the provided Cumulus workflows are internally self-referential._
```

Comment:
Definitely worth highlighting. Wonder if there's any way we can just say this once? Maybe a footnote or something else?

Reply:
I thought about that, but this *particular* note feels like we need to make the point of making a point.           That's just my opinion though, I'm willing to make it less pointed.

Reply:
I'm good with the repetition for the sake of clarity